jvb/src/test/kotlin/org/jitsi/videobridge/cc/allocation/BitrateControllerTest.kt

```diff
@@ -64,8 +64,15 @@ class BitrateControllerTest : ShouldSpec() {
      */
     override suspend fun beforeSpec(spec: Spec) = super.beforeSpec(spec).also {
         setNewConfig(
-            "videobridge.cc.bwe-change-threshold=0" +
-                "\n" + configWithMultiStreamDisabled, // Also disable multi stream support,
+            """
```

Comment:
Presumably the tests that use the BitrateController should expire them when they're done, both to avoid leaking the object and to test the expiration behavior?  (Maybe an explicit test for the expiration behavior?)